waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -522,6 +537,183 @@ protected String getUsage() {
         }
     }
 
+    /**
+     * Use {@code Producer} command to create a producer client that
+     * submits transactions to Waltz server.
+     *
+     * Producer process is closed once total sum of high watermarks for active
+     * partitions reaches num-callbacks.
+     */
+    private static final class CreateProducer extends ClientCliBaseClass {
+        private static final String NAME = "create-producer";
+        private static final String DESCRIPTION = "Creates one producer";
+
+        private CreateProducer(String[] args) {
+            super(args);
+        }
+
+        @Override
+        protected void configureOptions(Options options) {
+            Option txnPerClientOption = Option.builder("npt")
+                    .longOpt("num-produced-txn")
+                    .desc("Specify number of transactions to be produced")
+                    .hasArg()
+                    .build();
+            Option numClientsOption = Option.builder("nc")
+                    .longOpt("num-callbacks")
```

Comment:
After giving some thought I feel like making the producer to wait based on `num-callbacks` doesn't feel super intuitive. I was thinking instead we can add a CountDownLatch based on number of transactions that the producer has submitted to Waltz.

If you look at `HighWaterMarkTxnContext` which extends `TransactionContext`, we can implement the `onCompletion(boolean result)` method and based on if `result` is true which means the transaction has been submitted to Waltz successfully and the client will not resend it again. So, we can put a countdown here.

So, that way we can make the CreateProducer to only produce the transactions and terminate once it has successfully submitted instead of waiting on num producer callbacks.